doc/src/manual/linear-algebra.md

```diff
@@ -1,29 +1,133 @@
+  9.31908+0.0im
+ -2.40954+2.72095im
+ -2.40954-2.72095im
+
+julia> eigvecs(A)
+3×3 Array{Complex{Float64},2}:
+ -0.488645+0.0im  0.182546-0.39813im   0.182546+0.39813im
+ -0.540358+0.0im  0.692926+0.0im       0.692926-0.0im
+   0.68501+0.0im  0.254058-0.513301im  0.254058+0.513301im
+```
+
+In addition, Julia provides many [factorizations](@ref man-linalg-factorizations) which can be used to
+speed up problems such as linear solve or matrix exponentiation by pre-factorizing a matrix into a form
+more amenable (for performance or memory reasons) to the problem. See the documentation on [`factorize`](@ref)
+for more information. As an example:
+
+```jldoctest
+julia> A = [1.5 2 -4; 3 -1 -6; -10 2.3 4]
+3×3 Array{Float64,2}:
+   1.5   2.0  -4.0
+   3.0  -1.0  -6.0
+ -10.0   2.3   4.0
+
+julia> factorize(A)
+Base.LinAlg.LU{Float64,Array{Float64,2}}([-10.0 2.3 4.0; -0.15 2.345 -3.4; -0.3 -0.132196 -5.24947],[3,3,3],0)
+```
+
+Since `A` is not Hermitian, symmertric, triangular, tridiagonal, or bidiagonal, an LU factorization may be the
+best we can do. Compare with:
+
+```jldoctest
+julia> B = [1.5 2 -4; 2 -1 -3; -4 -3 5]
+3×3 Array{Float64,2}:
+  1.5   2.0  -4.0
+  2.0  -1.0  -3.0
+ -4.0  -3.0   5.0
+
+julia> factorize(B)
+Base.LinAlg.BunchKaufman{Float64,Array{Float64,2}}([-1.64286 0.142857 -0.8; 2.0 -2.8 -0.6; -4.0 -3.0 5.0],[1,2,3],'U',true,false,0)
+```
+
+Here, Julia was able to detect that `B` is in fact symmetric, and used a more appropriate factorization.
+Often it's possible to write more efficient code for a matrix that is known to have certain properties e.g.
+it is symmetric, or tridiagonal. Julia provides some special types so that you can "tag" matrices as having
+these properties. For instance:
+
+```jldoctest
+julia> B = [1.5 2 -4; 2 -1 -3; -4 -3 5]
+3×3 Array{Float64,2}:
+  1.5   2.0  -4.0
+  2.0  -1.0  -3.0
+ -4.0  -3.0   5.0
+
+julia> sB = Symmetric(B)
+3×3 Symmetric{Float64,Array{Float64,2}}:
+  1.5   2.0  -4.0
+  2.0  -1.0  -3.0
+ -4.0  -3.0   5.0
+```
+
+`sB` has been tagged as a matrix that's (real) symmetric, so for later operations we might perform on it,
+such as eigenfactorization or computing matrix-vector products, efficiencies can be found by only referencing
+half of it. For example:
+
+```jldoctest
+julia> B = [1.5 2 -4; 2 -1 -3; -4 -3 5]
+3×3 Array{Float64,2}:
+  1.5   2.0  -4.0
+  2.0  -1.0  -3.0
+ -4.0  -3.0   5.0
+
+julia> sB = Symmetric(B)
+3×3 Symmetric{Float64,Array{Float64,2}}:
+  1.5   2.0  -4.0
+  2.0  -1.0  -3.0
+ -4.0  -3.0   5.0
+
+julia> x = [1; 2; 3]
+3-element Array{Int64,1}:
+ 1
+ 2
+ 3
+
+julia> sB\x
+3-element Array{Float64,1}:
+ -1.73913
+ -1.1087
+ -1.45652
+```
+The `\\` operation here performs the linear solution. Julia's parser provides convenient dispatch
+to specialized methods for the *transpose* of a matrix left-divided by a vector, or for the various combinations
+of tranpose operations in matrix-matrix solutions. Many of these are further specialized for certain special
```

Comment:
transpose

it's not immediately clear from the example above why transposed left-divide is relevant here